### PR TITLE
Gift Entry: fix missing name from SGE header

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes
- Fixes missing donor name from SGE header when coming from an account/contact page
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by psadie-sfdo** [comment]
**lurch: attach W-039200

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HroBQAS" target="_blank">W-039200</a>

File: src/lwc/geFormRenderer/geFormRenderer.js [hunk 1]
```diff
@@ -43,7 +46,7 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     @api donorApiName;
     @api donorRecord;
 
-    fieldNames = [];
+    fieldNames = [ ACCOUNT_NAME_FIELD, CONTACT_NAME_FIELD ];
```
Comment (vmartinov-sforg) [inline src/lwc/geFormRenderer/geFormRenderer.js]: In getRecordFieldNames in utilTemplateBuilder.js we initialize the array of fields to 
let fieldNames = [`${apiName}.Id`]; 
to handle the case when the template does not contain account or contact field names and the form was not rendering. Can we get rid of that line now that we initialize the array with Contact.Name and Account.Name?

Comment (vmartinov-sforg) [reply src/lwc/geFormRenderer/geFormRenderer.js]: I'm working in another branch and the default template loads fine for me from contact/account. This bug was already fixed once.

Comment (vmartinov-sforg) [reply src/lwc/geFormRenderer/geFormRenderer.js]: This bug was already fixed once, and I don't see it happening in my branch. Did we regress maybe at some point?

**Pull request merged**